README.md

```diff
@@ -118,7 +118,7 @@ Let’s explain the plugin fields:
 * `influxDBToken` - the influxdb bucket token, the default value should be updated, copy it from InfluxDB site.
 * `influxDBFlushInterval` - its interval to send data to InfluxDB, the default value is 4000 (4 seconds).
 * `influxDBMaxBatchSize` - the max size of the batch with metrics, the default 2000 (2000 items of JMeter results).
-
+* `influxDBThresholdError` - the error threshold before stopping the import, the default value is 5. (@see `Important notes` for more detail.)
```

Comment:
let's use the markup language, please update the line like this

* `influxDBThresholdError` - the error threshold before stopping the import, the default value is 5. (see [Important notes](https://github.com/mderevyankoaqa/jmeter-influxdb2-listener-plugin#important-notes) for more detail.)

Reply:
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links to make the link for the https://github.com/mderevyankoaqa/jmeter-influxdb2-listener-plugin#important-notes